Client/core/CCore.cpp

```diff
@@ -184,7 +184,8 @@ CCore::~CCore(void)
     // Destroy tray icon
     delete m_pTrayIcon;
 
-    // Reset GTA volume levels by ignoring master level
+    // This will save GTA volume using the GTA volume level fader value set in settings
+    // and will not be "attenuated" by the master volume level setting.
```

Comment:
Can this be rephrased as

> This will set the GTA volume to the GTA volume value in the settings, and is not affected by the master volume setting.

?

Reply:
@qaisjp Sure! Done.

Reply:
oh i meant if that was what it meant. this works too though. thanks.

Reply:
Hehe. Yes, you were correct.
MTA specific settings would interfere with GTA and that's not really what we want.